grblas/base.py

```diff
@@ -82,7 +86,7 @@ def __call__(self, *optional_mask_and_accum, mask=None, accum=None, replace=Fals
         for key in optional_mask_and_accum:
             if isinstance(key, GbContainer):
                 mask_arg = key
-            elif type(key) is ComplementedMask:
+            elif type(key) in (ComplementedMask, StructuralMask, ComplementedStructuralMask):
```

Comment:
Small nit: I would use a set literal here.  I think Python is pretty good about making it fast.  A year or two ago I would have used a tuple as well.